index.html

```diff
@@ -1,29 +1,72 @@
-            rel="stylesheet"
-        />
-        <link
-            rel="stylesheet"
-            href="https://cdn.jsdelivr.net/npm/modern-normalize@1.1.0/modern-normalize.min.css"
-        />
-    </head>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Fresh Harvest Box!</title>
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" type="image/x-icon" href="/images/favicon.png" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=DM+Sans:ital,opsz,wght@0,9..40,100..1000;1,9..40,100..1000&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/modern-normalize@1.1.0/modern-normalize.min.css"
+    />
+  </head>
 
-    <body>
-        <!-- NA DOLE PRZYKŁAD JAK ZACZĄĆ DANĄ SEKCJĘ -->
-        <!-- CONTAINER JEST DLA KAŻDEJ SEKCJI OPRÓCZ MODALI -->
-        <section class="header">
-            <div class="container"></div>
-            <div class="mobile-menu"></div>
-        </section>
-    </body>
+  <body>
+    <!-- NA DOLE PRZYKŁAD JAK ZACZĄĆ DANĄ SEKCJĘ -->
+    <!-- CONTAINER JEST DLA KAŻDEJ SEKCJI OPRÓCZ MODALI -->
+    <section class="header">
+      <div class="container"></div>
+      <div class="mobile-menu"></div>
+    </section>
+
+    <section class="howitworks">
+      <div class="container">
+        <h2 class="howitworks-title h2-mobile">
+          To order your fruit basket, simply follow these easy steps
+        </h2>
+        <p class="p-all howitworks-text">
+          Our baskets are assembled with care and delivered straight to your
+          doorstep, so you can enjoy the taste of fresh fruit without ever
+          leaving your home. Whether you're looking for a healthy snack or a
+          thoughtful gift, our fruit baskets are the perfect choice.
+        </p>
+        <ul class="howitworks-fieldset">
+          <li class="howitworks-field">
+            <button class="howitworks-steps">First Step</button>
+            <p class="p-all howitworks-description">
+              Just choose the fruit you want to order by clicking on the
+              checkboxes next to it.
+            </p>
+          </li>
+          <li class="howitworks-field">
+            <button class="howitworks-steps">Second Step</button>
+            <p class="p-all howitworks-description">
+              Click on the basket and fill out the form.
+            </p>
+          </li>
+          <li class="howitworks-field">
+            <button class="howitworks-steps">Third Step</button>
+            <p class="p-all howitworks-description">
+              Sit back and relax! Your fresh fruit basket will be delivered.
+            </p>
+          </li>
+          <div class="howitworks-gif">
+            <img
```

Comment:
Test Test mamy tutaj jpg a powinnień być gif

Reply:
wiem, ale nie umiem skompresować gifa, nie mam teraz rano czasu, potem to zrobie